src/coreclr/jit/jiteh.cpp

```diff
@@ -1276,62 +1276,24 @@ void Compiler::fgSetHndEnd(EHblkDsc* handlerTab, BasicBlock* newHndLast)
 }
 
 //-------------------------------------------------------------
-// fgRebuildEHRegions: After reordering blocks, make EH regions contiguous
-// while maintaining relative block order, and update each region's end pointer.
+// fgFindTryRegionEnds: Walk the main method body, and set each try region's end block.
 //
-void Compiler::fgRebuildEHRegions()
+void Compiler::fgFindTryRegionEnds()
```

Comment:
This code isn't anything new. Since we no longer need to move cold EH blocks back in line after layout, we can bring back the EH repair logic I introduced in #108634. But now that we don't reorder handler regions, I've simplified this code to only reset try region entries in the main method body.